inc/core/patterns/bcgov-alpha3-title-and-links-video-bkgd.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Alpha V3 Title and Links with Video Background.
+ *
+ * @since 1.7.0
+ *
+ * @package Bcgov/Theme/Block
+ */
+
+ namespace Bcgov\Theme\Block;
+
+return [
+	'title'      => __( 'Title and Links, Video Background', 'bcgov_blocks_theme' ),
+	'categories' => [ 'bcgov-blocks-theme-general', 'bcgov-blocks-theme-alpha-v3' ],
+	'content'    => '<!-- wp:cover {"url":"' . esc_url( get_template_directory_uri() ) . '/assets/videos/clouds_optimised.mp4","id":2992,"dimRatio":0,"backgroundType":"video","minHeight":66,"minHeightUnit":"vh","contentPosition":"center center","isDark":false,"align":"full","style":{"spacing":{"margin":{"top":"0","bottom":"0"}}}} -->
```

Comment:
Very cool to use a sample video. Asset has a reasonably small file size. Looking forward to seeing this in action too.